modules/src/clients/ics11_beefy/client_def.rs

```diff
@@ -1,6 +1,6 @@
-use beefy_client::primitives::{ParachainHeader, ParachainsUpdateProof};
-use beefy_client::traits::ClientState as LightClientState;
 use beefy_client::BeefyLightClient;
+use beefy_client_primitives::ClientState as LightClientState;
```

Comment:
why not rename the struct?

Reply:
The struct comes from `beefy-rs` and in the context of the former it makes sense to call it `ClientState`.